File: chatwoot/php/src/CreateAContactExample.php
<?php

namespace OSEG\ChatwootExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Chatwoot;

$config = Chatwoot\Client\Configuration::getDefaultConfiguration();

$public_contact_create_update_payload = (new Chatwoot\Client\Model\PublicContactCreateUpdatePayload())
    ->setIdentifier(null)
    ->setIdentifierHash(null)
    ->setEmail(null)
    ->setName(null)
    ->setPhoneNumber(null)
    ->setAvatarUrl(null);

try {
    $response = (new Chatwoot\Client\Api\ContactsAPIApi(config: $config))->createAContact(
        inbox_identifier: null,
        data: public_contact_create_update_payload,
    );

    print_r($response);
} catch (Chatwoot\Client\ApiException $e) {
    echo "Exception when calling ContactsAPIApi#createAContact: {$e->getMessage()}";
}

File: chatwoot/ruby/src/CreateAContactExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
end

public_contact_create_update_payload = ChatwootClient::PublicContactCreateUpdatePayload.new

begin
    response = ChatwootClient::ContactsAPIApi.new.create_a_contact(
        "inbox_identifier_string", # inbox_identifier
        public_contact_create_update_payload, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling ContactsAPIApi#create_a_contact: #{e}"
end
